ci_framework/roles/devscripts/tasks/sub_tasks/12_yum_repo.yml

```diff
@@ -0,0 +1,35 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# codeready-builder repo is required for executing dev-scripts. This is the
+# final attempt in adding the repository
+
+# ToDo: Move these tasks to repo-setup
+- name: Enabling CRB repository
+  become: true
+  tags:
+    - bootstrap
+    - packages
+  when:
+    - cifmw_devscripts_crb_repo is defined
+    - not cifmw_devscripts_rhsm_enabled_repos
+  ansible.builtin.yum_repository:
```

Comment:
isn't that repository present, but just disabled, by default? Can't we just enable it instead?